apps/site/pages/components/organisms/product-shelf.mdx

```diff
@@ -0,0 +1,433 @@
+          {Array.from({ length: 5 }, (_, index) => (
+            <ProductShelfItem>
+              <ProductCard>
+                <ProductCardImage>
+                  <img
+                    data-fs-image
+                    src={product.image[0].url}
+                    alt={product.image[0].alternateName}
+                  />
+                </ProductCardImage>
+                <ProductCardContent
+                  title={product.isVariantOf.name}
+                  price={{
+                    value: product.offers.offers[0].price,
+                    listPrice: product.offers.offers[0].listPrice,
+                    formatter: useFormattedPrice,
+                  }}
+                />
+              </ProductCard>
+            </ProductShelfItem>
+          ))}
+        </ProductShelfItems>
+      </ProductShelf>
+    </OverviewSection>
+
+  </Tab>
+  <Tab>
+    ```tsx
+      <ProductShelf>
+        <ProductShelfItems>
+          {Array.from({ length: 5 }, (_, index) => (
+            <ProductShelfItem>
+              <ProductCard>
+                <ProductCardImage>
+                  <img
+                    data-fs-image
+                    src={product.image[0].url}
+                    alt={product.image[0].alternateName}
+                  />
+                </ProductCardImage>
+                <ProductCardContent
+                  title={product.isVariantOf.name}
+                  price={{
+                    value: product.offers.offers[0].price,
+                    listPrice: product.offers.offers[0].listPrice,
+                    formatter: useFormattedPrice,
+                  }}
+                />
+              </ProductCard>
+            </ProductShelfItem>
+          ))}
+        </ProductShelfItems>
+      </ProductShelf>
+    ```
+
+  </Tab>
+</Tabs>
+
+---
+
+## Props
+
+All `ProductShelf` related components support all attributes also supported by their respective HTML tags. Besides those attributes, the following props are also supported:
+
+### Product Shelf
+
+<ProductShelfPropsSection component="ProductShelf" />
+
+### Product Shelf Items
+
+<ProductShelfPropsSection component="ProductShelfItems" />
+
+### Product Shelf Item
+
+<ProductShelfPropsSection component="ProductShelfItem" />
+
+---
+
+## Design Tokens
+
+### Nested Elements
+
+#### Items
+
+<TokenTable>
+  <TokenRow
+    token="--fs-product-shelf-items-gap"
+    value="var(--fs-grid-gap-1)"
+  />
+  <TokenDivider />
+  <TokenRow
+    token="--fs-product-shelf-items-padding-top"
+    value="var(--fs-spacing-0)"
+  />
+  <TokenRow
+    token="--fs-product-shelf-items-padding-bottom"
+    value="var(--fs-spacing-3)"
+  />
+  <TokenDivider />
+  <TokenRow
+    token="--fs-product-shelf-items-left"
+    value="calc(-1 * var(--fs-spacing-1))"
+  />
+</TokenTable>
+
+---
+
+## Variants
+
+You can use [ProductCard variants](/components/molecules/product-card#variants) to create `ProductShelf` variations.
+
+### With Button
```

Comment:
Idk if we should add these variants in ProductShelf documentation instead of in ProductCard, because nothing changes in ProductGrid, just in ProductCard.

Reply:
It's just to show the possibilities